spec/xcpretty/parser_spec.rb

```diff
@@ -405,6 +405,20 @@ module XCPretty
         end
       end
 
+      it "parses uncaught exception errors" do
+        @formatter.should receive(:format_uncaught_exception).with(
+          'testRaisingUncaughtException',
+          'NSInvalidArgumentException',
+          '-[__NSPlaceholderDictionary initWithObjects:forKeys:count:]: attempt to insert nil object from objects[0]',
+          ['	0   CoreFoundation                      0x0000000109479a75 __exceptionPreprocess + 165',
+           '	1   libobjc.A.dylib                     0x0000000109112bb7 objc_exception_throw + 45',
+           '	2   CoreFoundation                      0x000000010938503f -[__NSPlaceholderDictionary initWithObjects:forKeys:count:] + 383',
+           '	3   CoreFoundation                      0x0000000109397d8b +[NSDictionary dictionaryWithObjects:forKeys:count:] + 59'])
```

Comment:
Line is too long. [132/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.